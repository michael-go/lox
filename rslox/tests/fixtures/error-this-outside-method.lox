print this; // At top level.

fun notMethod() {
  print this; // In a function.
} // TODO: getting an error also about this being unexpected.