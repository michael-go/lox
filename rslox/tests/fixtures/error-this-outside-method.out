# exit code: 65
# stdout:

# stderr:
[line 1] Error at 'this': Cannot use 'this' outside of a class.
[line 4] Error at 'this': Cannot use 'this' outside of a class.
[line 5] Error at '}': Expect expression.

